Cargo.toml

```diff
@@ -47,16 +47,17 @@ miden-tx              = { default-features = false, path = "crates/miden-tx", ve
 miden-tx-batch-prover = { default-features = false, path = "crates/miden-tx-batch-prover", version = "0.11" }
 
 # Miden dependencies
-assembly         = { default-features = false, package = "miden-assembly", version = "0.16.3" }
+assembly         = { branch = "next", default-features = false, git = "https://github.com/0xMiden/miden-vm.git", package = "miden-assembly", version = "0.18" }
 miden-crypto     = { default-features = false, version = "0.15.6" }
-miden-prover     = { default-features = false, version = "0.16.3" }
-miden-stdlib     = { default-features = false, version = "0.16.3" }
-miden-utils-sync = { default-features = false, version = "0.16.3" }
-miden-verifier   = { default-features = false, version = "0.16.3" }
-vm-core          = { default-features = false, package = "miden-core", version = "0.16.3" }
-vm-processor     = { default-features = false, package = "miden-processor", version = "0.16.3" }
+miden-prover     = { branch = "next", default-features = false, git = "https://github.com/0xMiden/miden-vm.git", version = "0.18" }
+miden-stdlib     = { branch = "next", default-features = false, git = "https://github.com/0xMiden/miden-vm.git", version = "0.18" }
+miden-utils-sync = { branch = "next", default-features = false, git = "https://github.com/0xMiden/miden-vm.git", version = "0.18" }
+miden-verifier   = { branch = "next", default-features = false, git = "https://github.com/0xMiden/miden-vm.git", version = "0.18" }
+vm-core          = { branch = "next", default-features = false, git = "https://github.com/0xMiden/miden-vm.git", package = "miden-core", version = "0.18" }
+vm-processor     = { branch = "next", default-features = false, git = "https://github.com/0xMiden/miden-vm.git", package = "miden-processor", version = "0.18" }
```

Comment:
Is there a reason to use `next` here? I think these should be just pointing to crates.io - i.e.:
```toml
assembly         = { default-features = false, package = "miden-assembly", version = "0.17" }
miden-crypto     = { default-features = false, version = "0.15.6" }
miden-prover     = { default-features = false, version = "0.17" }
miden-stdlib     = { default-features = false, version = "0.17" }
miden-utils-sync = { default-features = false, version = "0.17" }
miden-verifier   = { default-features = false, version = "0.17" }
vm-core          = { default-features = false, package = "miden-core", version = "0.17" }
vm-processor     = { default-features = false, package = "miden-processor", version = "0.17" }
```

Reply:
At the time, I wasn't aware 0.18 was released already. `0.17` is not sufficient.